src/main/cli/cli.c

```diff
@@ -3246,37 +3247,52 @@ static void cliBeeper(char *cmdline)
 }
 #endif
 
-#ifdef USE_RX_SPI
-void cliRxSpiBind(char *cmdline){
+#if defined(USE_RX_SPI) || defined (USE_SERIALRX_SRXL2)
+void cliRxBind(char *cmdline){
     UNUSED(cmdline);
-    switch (rxSpiConfig()->rx_spi_protocol) {
-    default:
-        cliPrint("Not supported.");
-        break;
+    if (featureIsEnabled(FEATURE_RX_SERIAL)) {
+        switch (rxConfig()->serialrx_provider) {
+        default:
+            cliPrint("Not supported.");
+            break;
+#if defined(USE_SERIALRX_SRXL2)
+        case SERIALRX_SRXL2:
+            srxl2Bind();
+            cliPrint("Binding SRXL2 receiver...");
+            break;
+#endif
+        }
+    } else if (featureIsEnabled(FEATURE_RX_SPI)) {
```

Comment:
This `else if` should be guarded by a `#if defined(USE_RX_SPI)` now.

Reply:
Done